app/models/image.rb

```diff
@@ -35,7 +35,7 @@ def decoded_image
 
   private
 
-  def image_size
+  def validate_image_size
```

Comment:
@lindgrenj6 should this be adding to the errors and not raising an exception. All the validators have to convey their results using the errors object, because we could have multiple validation errors and in the end when ActiveRecord sees that there is an errors object, it raises a single exception. By raising the exception I am not sure if you are short circuiting their logic.
```
errors.add(:image_size, "Image size exceeds max limit of #{MAX_IMAGE_SIZE.to_s(:human_size)}")
```

Reply:
Changed the exception that was thrown to just add to the errors array, it appears to still be working as expected.